src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -272,6 +272,8 @@ public function process_directives( string $html ): string {
 	 * it returns null if the HTML contains unbalanced tags.
 	 *
 	 * @since 6.5.0
+	 * @since 6.6.0 The function now adds a warning when the HTML contains unbalanced
+	 * tags or SVG/Math with directives.
```

Comment:
```suggestion
	 * @since 6.6.0 The function now adds a warning when the HTML contains unbalanced
	 *              tags or SVG/Math with directives.
```